adops/gam-creative-banner-sbs.md

```diff
@@ -0,0 +1,110 @@
+head_title: GAM Step by Step - Banner/Outstream/AMP Creatives
+description: Set up banner and outstream creatives for Prebid in Google Ad Manager.
+pid: 2
+sidebarType: 3
+---
+
+
+# GAM Step by Step - Banner/Outstream/AMP Creatives
+
+This page walks you through the steps required to create banner and outstream creatives to attach to your Prebid line items in Google Ad Manager (GAM).
+
+{: .alert.alert-success :}
+For complete instructions on setting up Prebid line items in Google Ad Manager, see [Google Ad Manager with Prebid Step by Step](/adops/step-by-step.html).
+
+
+1. In GAM, select **Delivery** > **Creatives**.
+2. Under the **Display creatives** tab, click **New Creative**.
+3. Select your advertiser, then click **Third party**.
+4. Enter a **Name** for your creative. For example, `Prebid – banner – 1x1`.
+5. Enter a **Target ad unit size** of `1x1`. This allows the creative to serve on all inventory sizes.
+
+{: .alert.alert-info :}
+These instructions assume you're using the Prebid Universal Creative (PUC). See the [Prebid Universal Creative](/overview/prebid-universal-creative.html) documentation for alternate approaches.
+
+{:start="6"}
+6. Select **Standard** as the **Code type**.
+
+{: .alert.alert-info :}
+**AMP**: If you are using AMP, you should still select **Standard** as the code type. The “AMP” option is for AMPHTML hosted by a 3rd party.
+
+{:start="7"}
+7. Enter one of the scripts shown below, depending on whether Prebid is configured to send all bids or only the top price bid.
+
+**Send All Bids Configuration**
+
+{: .alert.alert-warning :}
+Be sure to replace BIDDERCODE with the appropriate bidder. For example, if the bidder code is `BidderA`, the `adid` would be `%%PATTERN:hb_adid_BidderA%%`.
+
+```
+    <script src = "https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/creative.js"></script>
+    <script>
+      var ucTagData = {};
+      ucTagData.adServerDomain = "";
+      ucTagData.pubUrl = "%%PATTERN:url%%";
+      ucTagData.adId = "%%PATTERN:hb_adid_BIDDERCODE%%";
+      ucTagData.cacheHost = "%%PATTERN:hb_cache_host_BIDDERCODE%%";
+      ucTagData.cachePath = "%%PATTERN:hb_cache_path_BIDDERCODE%%";
+      ucTagData.uuid = "%%PATTERN:hb_cache_id_BIDDERCODE%%";
+      ucTagData.mediaType = "%%PATTERN:hb_format_BIDDERCODE%%";
+      ucTagData.env = "%%PATTERN:hb_env%%";
+      ucTagData.size = "%%PATTERN:hb_size_BIDDERCODE%%";
+      ucTagData.hbPb = "%%PATTERN:hb_pb_BIDDERCODE%%";
+      // mobileResize needed for mobile GAM only
+      ucTagData.mobileResize = "hb_size:%%PATTERN:hb_size_BIDDERCODE%%";
+      try {
+        ucTag.renderAd(document, ucTagData);
+      } catch (e) {
+        console.log(e);
+      }
+    </script>
+```
+
+{: .alert.alert-danger :}
+Warning: Be sure none of the attribute names are longer than 20 characters. See [Send All Bids Key Value Pairs](/adops/send-all-vs-top-price.html#key-value-pairs) for more information.
+
+**Send Top Price Bid Configuration**
+
```

Comment:
Suggest adding "In top-price mode, you can make use of the GAM `TARGETINGMAP` feature instead of listing out each attribute."